data-source/jsonnet/northern-ireland/blocks/individual/qualifications/a_level.jsonnet

```diff
@@ -1,9 +1,9 @@
 local placeholders = import '../../../../common/lib/placeholders.libsonnet';
 local rules = import '../../../../common/lib/rules.libsonnet';
 
-local nonProxyTitle = 'Have you achieved an AS, A level or equivalent qualification?';
+local nonProxyTitle = 'Have you achieved an A level, AS level or equivalent qualifications?';
```

Comment:
I don't understand why this/subsequent questions have changed to 'qualifications' (plural)? Were they originally incorrectly worded?

Reply:
🤷‍♂ I pointed it out only cause card states `qualifications`.

Reply:
It feels a bit off, as the preceding items are singular. But it could be argued that it may be- as it is in the case of an AS level, that you might need more than one qual to form equivalence i.e. two AS levels for one A level.

The or creates a break in the sentence allowing for a plurality disagreement, I *think*.

But it is what was requested, as Mebin correctly noticed.

Reply:
Ah, fair enough - difficult to see which card this relates to given its not linked to here.

Reply:
Ah sorry, thought most people came via Trello. https://trello.com/c/JrPo6wKV/3034-nisra-content-tweaks